tests/functional/spec/features/default_value/default_value.spec.js

```diff
@@ -0,0 +1,20 @@
+const helpers = require('../../../helpers');
+
+const QuestionPage = require('../../../pages/features/default_value/number-question.page.js');
+const Summary = require('../../../pages/features/default_value/summary.page.js');
+
+describe('@watch Feature: Default Value', function() {
```

Comment:
Remove `@watch`. Also small nit pick, but for our functional specs don't we use hyphens not underscores.